src/incubator/WheelPicker/index.tsx

```diff
@@ -143,7 +159,7 @@ const WheelPicker = React.memo(
       [itemHeight]
     );
 
-    const separators = () => {
+    const renderCentralFrame = () => {
```

Comment:
Sorry to nudge you 😬
`renderSeparatorsAndLabel` or something, make more sense - at least to me, didn't get at first what a central frame is..

Reply:
Done.
I also changed the screen a bit to make it fit for small screens.
verified on Android as well